indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/SimpleResourceManagementConfig.java

```diff
@@ -99,6 +99,7 @@ public SimpleResourceManagementConfig setWorkerVersion(String workerVersion)
     return this;
   }
 
+  // Do not use this if possible. Assuming all workers will have the same port is bad for containers.
```

Comment:
Since there is no established alternative, and one worker per IP is assumed in a lot of places, not yet. Once that assumption is removed more generally, this can be deprecated.